lib/collection/src/shards/local_shard/mod.rs

```diff
@@ -108,6 +108,7 @@ pub struct LocalShard {
 
 /// Shard holds information about segments and WAL.
 impl LocalShard {
+    /// Moves `wal`, `segments` and `clocks` data from one path to another.
     pub async fn move_data(from: &Path, to: &Path) -> CollectionResult<()> {
```

Comment:
In `move_data`, lets move the initializing file marker as well if it exists. Because after moving, we should still consider it to be not initialized.

Reply:
One issue with this approach is that the marker file does not live inside the shard folder but outside.

Reply:
Would it be worth to move it inside, to support this?